### ISSUE TITLE
Key repeat in hyper canary

### ISSUE THREAD

**Issue by pablopunk** [zeit/hyper#2297]
- [ ] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**:  MacOS 12.13 High Sierra
- **Hyper.app version**: 2.0.4
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/pablopunk/5af1eb66a0ecd9833a252a63d38c3547
- **Relevant information from devtools** _(CMD+ALT+I on Mac OS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: Canary straight out of the box

## Issue
Basically pressing and holding any letter won't repeat it. Like, I press and hold `k` and it prints just one.

Also, if I press and hold `space` it does repeat this one.

**Comment by derrickpelletier** [comment]
Issue originates in `term.js`: https://github.com/sourcelair/xterm.js/issues/265

**Comment by albinekb** [comment]
Here's a workaround: https://stackoverflow.com/questions/39972335/how-do-i-press-and-hold-a-key-and-have-it-repeat-in-vscode

While we investigate the best way to solve this.

**Comment by JesseLeite** [comment]
>defaults write co.zeit.hyper ApplePressAndHoldEnabled -bool false

Doesn't seem to work for me :(

**Comment by pablopunk** [comment]
I fixed it for all apps on my mac:

```sh
defaults write NSGlobalDomain ApplePressAndHoldEnabled -bool false
```

Then reboot and it's done!

**Comment by ipatch** [comment]
@pablopunk thanks for sharing, do you know of any way for the below command to take affect without a reboot?
```
defaults write NSGlobalDomain ApplePressAndHoldEnabled -bool false
```

**Comment by pablopunk** [comment]
@ipatch not sure, maybe logout and login, try that.

**Comment by SavePointSam** [comment]
looks like xterm recently fixed this problem on their end

**Comment by autonome** [comment]
> looks like xterm recently fixed this problem on their end

Fixed in 3.10.0 of xterm last December in xtermjs/xterm.js#265

Hyper is on 3.12.0 as of #3536.

But the problem still exists for me in Hyper 2.1.2. Maybe not in stable yet? How can I find that out?
